frontend/public/features.ts

```diff
@@ -82,27 +90,30 @@ const detectOpenShift = dispatch => coFetchJSON(openshiftPath)
       : handleError(err, FLAGS.OPENSHIFT, dispatch, detectOpenShift)
   );
 
-const detectCanListNS = dispatch => {
-  const req = {
-    spec: {
-      resourceAttributes: {
-        name: 'namespaces',
-        verb: 'list',
-      },
-    },
-  };
-  k8sCreate(SelfSubjectAccessReviewModel, req)
-    .then(
-      res => setFlag(dispatch, FLAGS.CAN_LIST_NS, res.status.allowed),
-      err => handleError(err, FLAGS.CAN_LIST_NS, dispatch, detectCanListNS)
-    );
+const detectSSARRestrictions = dispatch => {
+  [
+    [ FLAGS.CAN_LIST_NS,    { spec: { resourceAttributes: { name: 'namespaces', verb: 'list' } } }],                // eslint-disable-line no-multi-spaces
+    [ FLAGS.CAN_LIST_NODE,  { spec: { resourceAttributes: { name: 'nodes', verb: 'list' } } }],                     // eslint-disable-line no-multi-spaces
+    [ FLAGS.CAN_LIST_PVC,   { spec: { resourceAttributes: { name: 'persistentvolumeclaims', verb: 'list' } } }],    // eslint-disable-line no-multi-spaces
+    [ FLAGS.CAN_LIST_STORE, { spec: { resourceAttributes: { name: 'storageclasses', verb: 'list' } } }],            // eslint-disable-line no-multi-spaces
+    [ FLAGS.CAN_LIST_CRD,   { spec: { resourceAttributes: { name: 'customresourcedefinitions', verb: 'list' } } }], // eslint-disable-line no-multi-spaces
```

Comment:
This was my bug originally, but this should be `resource` instead of `name` in these requests.